api/cryptocom/rest_action_test.go

```diff
@@ -0,0 +1,276 @@
+package cryptocom
+
+import (
+  "bytes"
+  "database/sql"
+  "encoding/json"
+  "errors"
+  "fmt"
+  "io"
+  "net/http"
+  "net/http/httptest"
+  "testing"
+
+  "github.com/bmizerany/assert"
+)
+
+/* global variable declaration */
+var mockRestRootURL string = "restRootURL"
+
+type mockHTTPClient struct {
```

Comment:
https://golang.org/pkg/net/http/httptest/

Reply:
Can you expand? I think I already use it in ```server   *httptest.Server``` and I mock response in function ```testRest```

Reply:
testing with httptest do not need any mocked clients, you only need to run the server in your spec and set its' URL as client restURL.
Check this example https://golang.org/pkg/net/http/httptest/#example_Server

Reply:
Ok removed